deploy/kube-bench/values.yaml

```diff
@@ -0,0 +1,27 @@
+# Default values for kube-bench.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+image:
+  repository: aquasec
+  name: kube-bench
+  tag: latest
+
+# runConfig is the configuration for running kube-bench
+runConfig:
+
+  # version is the Kubernetes version `--version` argument passed to kube-bench
+  # which is mandatory when run from inside a container since version
+  # auto-detection only works when kube-bench is run directly on the node.
+  version: 1.13
```

Comment:
this can be automated using `. Capabilities.KubeVersion`. it's ok if you prefer to leave it for another PR, just noting it here.

Reply:
I have not added this since the latest version supported by kube-bench could be different from the latest version of Kubernetes that a user could have installed.